rotkehlchen/db/upgrades/v31_v32.py

```diff
@@ -0,0 +1,33 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures import HistoryBaseEntry
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def upgrade_v31_to_v32(db: 'DBHandler') -> None:
+    """Upgrades the DB from v31 to v32
+
+    After changing the HistoryBaseEntry the identifiers for the events
```

Comment:
Question that I also asked myself when working on the events decoding for eth transactions. Why not just generate a random PK identifier and return it when saving in the DB?